manifests/m/Microsoft/WindowsTerminal/1.17.11461.0/Microsoft.WindowsTerminal.installer.yaml

```diff
@@ -0,0 +1,44 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.WindowsTerminal
+PackageVersion: 1.17.11461.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.19041.0
+Scope: user
+InstallModes:
+- silent
+Commands:
+- wt
+PackageFamilyName: Microsoft.WindowsTerminal_8wekyb3d8bbwe

```

Comment:
```suggestion
PackageFamilyName: Microsoft.WindowsTerminal_8wekyb3d8bbwe
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.UI.Xaml.2.7
```

Reply:
In a sandbox environment, this package also requires VCLibs. We would have to do something like

```
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCLibs.Desktop.14
    MinimumVersion: 14.0.30704.0
  - PackageIdentifier: Microsoft.UI.Xaml.2.7
    MinimumVersion: 7.2208.15002.0
```